platform/server/http/Router/index.ts

```diff
@@ -5,9 +5,9 @@ import { Headers, filterHeaders } from './headers';
 import { ObjectSetting, Props, Any, TypeOf } from '../../../lib/schema';
 
 export interface Route<
-  Params extends ObjectSetting<{}>,
-  Query extends ObjectSetting<{}>,
-  Body extends ObjectSetting<{}>
+  Params extends ObjectSetting<any>,
+  Query extends ObjectSetting<any>,
+  Body extends ObjectSetting<any>
```

Comment:
This was also because of the same express issue, probably should have a TODO, too.

Reply:
This actually wasn't because of express, but because of stricter variance checking in TypeScript 2.6. Basically that `ObjectSetting<{}>` cannot be assigned to e.g. `ObjectSetting<Params>` further down in the same file. There might be ways of writing that type better though, I just didn't have any other immediate ideas.

(Btw, https://flow.org/en/docs/lang/variance/ is a good explanation of variance)